src/stores/pacbioPoolCreate.js

```diff
@@ -551,11 +543,9 @@ export const usePacbioPoolCreateStore = defineStore('pacbioPoolCreate', {
       const request = rootStore.api.traction.pacbio.pools
       const promise = request.create({
         data: payload({ used_aliquots, pool }),
-        include: 'tube',
       })
-      const { success, body: { included = [] } = {}, errors } = await handleResponse(promise)
-      const { tubes: [tube = {}] = [] } = groupIncludedByResource(included)
-      const { attributes: { barcode = '' } = {} } = tube
+      const { success, body: { data = {} } = {}, errors } = await handleResponse(promise)
```

Comment:
_Groups the included resources by type and extracts the barcode from the first tube_. - this can be removed from the comment section on 'createPool' method